modules/lvms-troubleshooting-performing-a-forced-cleanup.adoc

```diff
@@ -66,30 +70,30 @@ $ oc get lvmvolumegroup
 No resources found
 ----
 
-.. If there are any `LVMVolumeGroup` CRs left, remove their finalizers by running the following command:
+.. If the `LVMVolumeGroup` CRs are present, remove their finalizers by running the following command:
 +
 [source,terminal]
 ----
 $ oc patch lvmvolumegroup <name> -p '{"metadata":{"finalizers":[]}}' --type=merge <1>
 ----
-<1> Replace `<name>` with the name of the CR.
+<1> Replace `<name>` with the name of the `LVMVolumeGroup` CR.
 
-.. After removing their finalizers, delete the CRs by running the following command:
+.. After removing the finalizers, delete the `LVMVolumeGroup` CRs by running the following command:
 +
 [source,terminal]
 ----
 $ oc delete lvmvolumegroup <name> <1>
 ----
-<1> Replace `<name>` with the name of the CR.
+<1> Replace `<name>` with the name of the `LVMVolumeGroup` CR.
 
-. Remove any `LVMVolumeGroupNodeStatus` CRs by running the following command:
+. Delete any `LVMVolumeGroupNodeStatus` CRs by running the following command:
 +
 [source,terminal]
 ----
 $ oc delete lvmvolumegroupnodestatus --all
```

Comment:
I think all objects (except LogicalVolume here) are namespaced, meaning it should be run with a project configured or with a --namespace

Reply:
The first step in this proc is:

Switch to the `openshift-storage` namespace by running the following command:
`$ oc project openshift-storage`

Considering this step, will the other commands without "-n <namespace" work fine?

Reply:
Ah my bad, yes with that in mind, we are good here